apps/docs/data/components/searchfield/SearchFieldH5.tsx

```diff
@@ -0,0 +1,21 @@
+import {SearchField} from '@comfortdelgro/react-compass'
+
+function H5Example() {
+  return (
+    <div style={{gap: '1rem', display: 'flex'}}>
+      <SearchField
+        placeholder='Search'
+        onSubmit={(text) => console.log(text)}
+        isH5Variant
+      />
+      <SearchField
+        isDisabled
+        placeholder='Search'
+        isH5Variant

```

Comment:
```suggestion
        h5
```

I think just put h5 for consistency with other components